src/nvim/eval.lua

```diff
@@ -1373,6 +1373,8 @@ M.funcs = {
       		no item is selected when using the <Up> or
       		<Down> keys)
          inserted	Inserted string. [NOT IMPLEMENTED YET]
+         preview_winid     Info floating preview window id.
```

Comment:
any good name of `preview_` prefix ? seems like a little long. doc format not looks good.

Reply:
it is fine as it is.